aiidalab_widgets_base/codes.py

```diff
@@ -2,7 +2,30 @@
 
 from __future__ import absolute_import
 import ipywidgets as ipw
+
+from subprocess import check_output
 from IPython.display import clear_output
+from aiida import load_dbenv, is_dbenv_loaded
+from aiida.backends import settings
+if not is_dbenv_loaded():
```

Comment:
We are intentionally avoiding `load_dbenv` at module import in the other widgets.
Why should we break this here?

Note that the `__init__.py` imports this file.

Reply:
ok, as we agreed I will replace `load_dbenv` function with [with_dbenv](https://github.com/aiidateam/aiida_core/blob/aaa8c687bd325bedf1d4aeca479d4ca9a2b96eb4/aiida/cmdline/dbenv_lazyloading.py#L38-L56) decorator on `__init__()` function of each class.

Reply:
ok, so we change the final decision. This code will be put in the top `__init__.py` file of the module:
```
from aiida import load_dbenv, is_dbenv_loaded
from aiida.backends import settings
if not is_dbenv_loaded():
    load_dbenv(profile=settings.AIIDADB_PROFILE)
```